tests/integration/test_notebooks.py

```diff
@@ -74,3 +82,7 @@ def _exec_notebook(notebook_filename):
 @pytest.mark.parametrize("notebook_file", get_notebooks())
 def test_notebook(notebook_file):
     _exec_notebook(notebook_file)
+
+
+def test_has_notebooks():
+    assert len(get_notebooks()) >= 35, "there are probably some notebooks that were not discovered"
```

Comment:
does it need to be this high? would 10 do? i guess it would mean this test will fail less frequently

Reply:
also can you make the text more descriptive? I'd find it a little hard to debug that error message

Reply:
this test would only fail if we delete some notebooks, I think it's suitably obvious why the test would fail in that case...:)

Reply:
`35` looks like a random number, and that is a random number.